### PR TITLE
Hotfix: Change path name for get file Android

### PR DESCRIPTION
**Pull request by nqhhdev**

### CODE REVIEW

File: lib/utils/matrix_sdk_extensions/matrix_file_extension.dart [hunk 1]
```diff
@@ -11,9 +11,8 @@ import 'package:share_plus/share_plus.dart';
 import 'package:fluffychat/utils/platform_infos.dart';
 import 'package:fluffychat/utils/size_string.dart';
 
-const String fileName = "/storage/emulated/0/Download";
-
 extension MatrixFileExtension on MatrixFile {
+  static const String _downloadDirectoryPath = "/storage/emulated/0/Download";
```
Comment (hoangdat) [inline lib/utils/matrix_sdk_extensions/matrix_file_extension.dart:15]: maybe it only works with emulator, not work with real device,

File: lib/utils/matrix_sdk_extensions/matrix_file_extension.dart [hunk 2]
```diff
@@ -44,7 +43,7 @@ extension MatrixFileExtension on MatrixFile {
   }
 
   Future<Directory> getDownloadDirectoryAndroid() async {
-    final defaultDownloadDirectory = Directory(fileName);
+    final defaultDownloadDirectory = Directory(_downloadDirectoryPath);
```
Comment (hoangdat) [inline lib/utils/matrix_sdk_extensions/matrix_file_extension.dart:46]: should use `getDirectory....`

File: lib/utils/matrix_sdk_extensions/matrix_file_extension.dart (hunk 1 above)
Comment (sherlockvn) [reply lib/utils/matrix_sdk_extensions/matrix_file_extension.dart]: Why don't you use path_provider package to get path ?

**Review by sherlockvn — APPROVED** [review]

**Review by drminh2807 — APPROVED** [review]

**Comment by hoangdat** [comment]
@nqhhdev why we need this fix? what problem?

**Review by hoangdat — APPROVED** [review]

**Review by dab246 — APPROVED** [review]
lgtm

**Pull request merged**